src/commands/config/set.js

```diff
@@ -84,6 +90,10 @@ ConfigSet.flags = {
     char: 'e',
     description: 'Sets an Edge configuration.',
   }),
+  'require-profile-input': flags.boolean({
+    description: 'Whether the profile flag for Twilio CLI commands is required or not.',
+    allowNo: true,
+  }),
```

Comment:
[Q] Do we require default to `false` here?

Reply:
It would change the functionality for the `require-profile-input` flag. Right now the we only need to pass the flag when we try to update it. Consider the scenario when the flag is set. Now if the default is set to false, whenever the config:set command is called for some other property say `edge`. it would try to override the `require-profile-input` flag to false, even if the flag is not passed.

Reply:
Got it, makes sense. :)